SnapshotTests/SnapshotTestCase.swift

```diff
@@ -189,3 +189,38 @@ extension Snapshotting where Value == UIView, Format == UIImage {
         )
     }
 }
+
+extension Snapshotting where Value == UIViewController, Format == UIImage {
+    static var extra3LargeFontStrategyLandscape: Self {
+        let traits = UITraitCollection(traitsFrom: [
+            .init(forceTouchCapability: .available),
+            .init(layoutDirection: .leftToRight),
+            .init(preferredContentSizeCategory: .medium),
+            .init(userInterfaceIdiom: .phone),
+            .init(horizontalSizeClass: .regular),
+            .init(verticalSizeClass: .compact),
+            .init(preferredContentSizeCategory: .accessibilityExtraExtraExtraLarge)
+        ])
+        let safeArea: UIEdgeInsets = .init(top: 0, left: 47, bottom: 21, right: 47)
+        let size: CGSize = .init(width: 844, height: 390)
+        let viewImageConfig: ViewImageConfig = .init(safeArea: safeArea, size: size, traits: traits)
+
+        return Self.image(on: viewImageConfig)
+    }
+
+    static var imageLandscape: Self {
```

Comment:
It seems like most of the trait collection items are the same, so we can put them in one reusable list:
```
private static var commonTraitCollection: [UITraitCollection] = [
        .init(forceTouchCapability: .available),
        .init(layoutDirection: .leftToRight),
        .init(preferredContentSizeCategory: .medium),
        .init(userInterfaceIdiom: .phone),
        .init(horizontalSizeClass: .regular),
        .init(verticalSizeClass: .compact),
    ]

static var extra3LargeFontStrategyLandscape: Self {
        let traits = UITraitCollection(traitsFrom: [
            .init(preferredContentSizeCategory: .accessibilityExtraExtraExtraLarge)
        ] + commonTraitCollection)
/// .....

static var imageLandscape: Self {
        let traits = UITraitCollection(traitsFrom: commonTraitCollection)
/// .....

```

Reply:
fixed [3a0d35b](https://github.com/salemove/ios-sdk-widgets/pull/774/commits/3a0d35bd1e04d360f1a3c9b8a91ba1582c231bdc)